tests/test_notebook.py

```diff
@@ -76,7 +76,9 @@ def load_module(self, fullname):
                         code = re.sub(r'n_samples_tsne = \d+', "n_samples_tsne = 10", code)
                         code = re.sub(r'n_samples_posterior_density = \d+', "n_samples_posterior_density = 2", code)
                         code = re.sub("save_path = 'data/'", "save_path = '"+os.getcwd()+"'", code)
+                        code = re.sub('save_path = "data/"', "save_path = '"+os.getcwd()+"'", code)
```

Comment:
is that a duplicate line ?

Reply:
This line was added on purpose. 
The notebook tests failed locally because the regex was looking for ... = 'data/' and it was ... = "data/" in the code.
There are no best practices in Python regarding simple quotes or double quotes, the tests will now account for both practices.

Reply:
That looks dangerous. Ok let's stick to that, thanks for pointing it out